dashboard/modules/metrics/grafana_dashboard_factory.py

```diff
@@ -292,26 +292,38 @@ def max_plus_pending(max_resource, pending_resource):
     Panel(
         id=34,
         title="Node Memory by Component",
-        description="The physical (hardware) memory usage across the cluster, broken down by component. This reports the summed USS (unique set size) per Ray component. Ray components consist of system components (e.g., raylet, gcs, dashboard, or agent) and the method names of running tasks/actors.",
+        description="The physical (hardware) memory usage across the cluster, broken down by component. This reports the summed RSS-SHM per Ray component, which corresponds to the heap memory usage. Ray components consist of system components (e.g., raylet, gcs, dashboard, or agent) and the process (that contains method names) names of running tasks/actors.",
```

Comment:
RSS also includes stack memory usage I believe? If we are reporting RSS - SHM, then it's also not "physical (hardware) memory usage"? This might be more accurate as "an approximate memory usage per proc"?  

 
Also, does this help? We couldn't get it?  https://linux.die.net/man/8/smem

Reply:
Actually - this should have USS? https://psutil.readthedocs.io/en/latest/#psutil.Process.memory_full_info ?

Reply:
> I failed to find the exact way to calculate uss unfortunately… 

Isn't this why we are not using uss? But https://psutil.readthedocs.io/en/latest/#psutil.Process.memory_full_info contains the USS info directly.

Reply:
hmm maybe the description wasn't clear. The reason why we don't want to use USS is because it contains the shm. And shm usage doesn't decrease although the worker doesn't use objects because of optimization we have.

When I say "don't know how to calculate USS", I meant how exactly this is calculated from the systems (so I couldn't figure out why it contains shm usage sometimes)